platform/mbed_mpu_mgmt.h

```diff
@@ -35,6 +35,8 @@ extern "C" {
 
 #define mbed_mpu_manager_init() mbed_mpu_init()
 
+#define #define mbed_mpu_manager_deinit() mbed_mpu_free()
```

Comment:
Uh-oh.

Reply:
At least is isn't #undefined behavior. Just kidding, I pushed a fix.